cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -78,23 +83,28 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
     return std::make_shared<arrow::Buffer>("");
   }
 
+  /// Read the byte stream. Returns a buffer of the objects in the stream.
   arrow::Result<std::shared_ptr<arrow::Buffer>> Read(int64_t nbytes) {
     ARROW_ASSIGN_OR_RAISE(auto buffer, ReadAt(pos_, nbytes));
     pos_ += buffer->size();
     return std::move(buffer);
   }
 
+  /// Read the byte stream into an output stream. Returns read data from the input.
```

Comment:
Read a specified number of bytes from the current position into an output stream

Reply:
Done.